gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManagerOptions.java

```diff
@@ -29,14 +30,14 @@
   private static final long serialVersionUID = 538303101192527452L;
   private static final String GCRM_SCOPE = "https://www.googleapis.com/auth/cloud-platform";
   private static final Set<String> SCOPES = ImmutableSet.of(GCRM_SCOPE);
+  private static final String DEFAULT_HOST = "https://cloudresourcemanager.googleapis.com";
```

Comment:
Unsure, I got that URL from the docs: https://cloud.google.com/resource-manager/reference/rest/